scripts/mysql_system_users.sql

```diff
@@ -78,7 +78,10 @@ REVOKE ALL PRIVILEGES, GRANT OPTION FROM 'mysql.pxc.internal.session'@localhost;
 
 -- Due to bugs with roles, we need to grant superuser access here
 GRANT ALL PRIVILEGES ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
-GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER, SHUTDOWN,CLONE_ADMIN, SYSTEM_USER,GROUP_REPLICATION_STREAM  ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
+-- Needed by sst_xtrabackup
+GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
+-- Additionally needed by sst_clone
+GRANT SHUTDOWN,CLONE_ADMIN, SYSTEM_USER,GROUP_REPLICATION_STREAM  ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
```

Comment:
Why is `GROUP_REPLICATION_STREAM` privilege needed for clone?

Reply:
TBH I do not know why is there, I have no memory of it, but was not in before so for some reason I had add it, but it is not used by clone. that is gr only

Reply:
So, can we remove it?

Reply:
Not needed. Removed.